components/malloc_size_of/lib.rs

```diff
@@ -720,6 +720,7 @@ malloc_size_of_is_0!(std::time::Duration);
 malloc_size_of_is_0!(std::time::Instant);
 malloc_size_of_is_0!(std::time::SystemTime);
 malloc_size_of_is_0!(style::font_face::SourceList);
+malloc_size_of_is_0!(style::queries::values::PrefersColorScheme);
```

Comment:
So this change is because `PrefersColorScheme` lost the derived `MallocSizeOf` during the rebase. Was that intentional?

Reply:
It lost it in the upstreaming. I didn't include it in the upstreamed patch because Stylo doesn't use it (and Gecko's version upstream didn't include the derive). Realised that Servo does when syncing. Figured that the easiest thing to do was the fix it here seeing as Servo has it's own version of `malloc_size_of` now.